src/Microsoft.AspNetCore.Mvc.Core/Internal/FileResultExecutorBase.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.AspNetCore.Mvc.Internal
+{
+    public class FileResultExecutorBase
+    {
+        public FileResultExecutorBase(ILogger logger)
+        {
+            Logger = logger;
+        }
+
+        protected ILogger Logger { get; }
```

Comment:
`private` unless I'm missing something. A `private` field would also be fine but I think you read my comments out of order (sorry) :sob: Not worth going back to a field unless you feel more strongly about it than I do.